DigitalLearningSolutions.Web/ViewModels/MyAccount/MyAccountEditDetailsFormData.cs

```diff
@@ -62,5 +77,84 @@ protected MyAccountEditDetailsFormData(MyAccountEditDetailsFormData formData)
         public IFormFile? ProfileImageFile { get; set; }
 
         public bool IsDelegateUser { get; set; }
+
+        [RunValidateMethod(nameof(ValidateProfessionalRegistrationNumber))]
+        public string? ProfessionalRegistrationNumber { get; set; }
+
+        [RunValidateMethod(nameof(ValidateHasProfessionalRegistrationNumber))]
+        public YesNoSelectionEnum HasProfessionalRegistrationNumber { get; set; }
+
+        public IEnumerable<ValidationResult> ValidateHasProfessionalRegistrationNumber()

```

Comment:
Similar to the other validate method, this will be wanted by the registration journey as well. I think it's right to make it an attribute of its own.

Reply:
See other comment.